api/src/app.ts

```diff
@@ -235,6 +240,17 @@ export const build = async (
       await fastify.register(authRoutes);
     }
   });
+
+  if (FREECODECAMP_NODE_ENV !== 'production') {
```

Comment:
Shouldn't this be "is staging" or have its own env var?

Reply:
I asked you if it should have its own env var, and you said "no", because `FREECODECAMP_NODE_ENV` perfectly suits the bill.

Reply:
I might have, but nonetheless: why only in development? Isn't the plan to put this on staging first?

Reply:
Currently, there is no plan with deploying this section of code. At least not one that is needed for this PR. The only thing is it must NOT go to production.

What is `FREECODECAMP_NODE_ENV` on `.dev`? Is it `"production"` or `"staging"`?

Reply:
`FREECODECAMP_NODE_ENV` is `development` by default locally and `production` on the production sites (.org and .dev). If you want to distinguish between .org and .dev, `DEPLOYMENT_ENV` does that for you.

If the idea is only allow this locally, please add a comment so we know this isn't intended to be used yet.